src/test/java/no/nav/pto/veilarbportefolje/oppfolging/VeilederTilordnetServiceTest.java

```diff
@@ -45,6 +45,24 @@ void skal_oppdatere_tilordnet_veileder() {
         assertThat(bruker.isNy_for_veileder()).isTrue();
     }
 
+    @Test
+    void skal_oppdatere_tilordnet_veileder_med_null() {
+        final AktorId aktoerId = randomAktorId();
+        final VeilederId nyVeileder = VeilederId.of(null);
```

Comment:
Are you sure that it wont be final VeilederId nyVeileder = null ?

Reply:
VeilederId will be created, because of this line: https://github.com/navikt/veilarbportefolje/blob/dev/src/main/java/no/nav/pto/veilarbportefolje/domene/value/VeilederId.java#L9

Reply:
ahh ok 🎉

Reply:
I will write test for that one, to be sure

Reply:
hm, you are right..

Reply:
@stianStensli I fixed that one, can you pls check again